package.json

```diff
@@ -11,9 +11,11 @@
     "test": "npx turbo test --concurrency=100%",
     "fmt": "npx prettier --write \"**/*.{js,ts,tsx,md,json}\"",
     "docs:build": "mdbook build book",
-    "docs:serve": "mdbook serve book"
+    "docs:serve": "mdbook serve book",
+    "generate:snippets": "bluehawk snip --output book/generated-snippets/ts teams.ts --ignore packages/graph/** --ignore !**/*.ts"
```

Comment:
nitpick: could we do "gen:snippets"? hahaha I am a lazy typer

Reply:
ofc